charts_web/android/app/build.gradle

```diff
@@ -0,0 +1,68 @@
+def localProperties = new Properties()
```

Comment:
Since this is for charts_web I think we can remove `android`, `ios` and `macos` folders